src/store/talkHashStore.js

```diff
@@ -22,6 +22,8 @@
 
 import { showError, TOAST_PERMANENT_TIMEOUT } from '@nextcloud/dialogs'
 
+const talkBroadcastChannel = new BroadcastChannel('nextcloud:talk')
+
```

Comment:
It works fine and is totally my personal preference, but I'd also move this to a separated module, so we won't need to create it again each time and bother about typos in the channel name.